server/gql/schema.js

```diff
@@ -124,13 +126,24 @@ const upsertResolver = async ({
         requestMapping,
         responseMapping,
         type,
-        field
+        field,
+        publish: publisher
     };
 
     let result;
     if (id) {
         const updated = await resolver.findById(id);
         result = updated.update(properties);
+        if (properties.publish) {
+            const subExists = await subscription.findOne({ where: { field: properties.publish } });
```

Comment:
Looks good, but maybe extract it into a new function like `checkAndUpdateSubscriptions`

Reply:
In 2 functions check and update.  It is not a good practice function as credateDeleteUpdate. Each one should be responsible for one action.

Reply:
@pb82 @camilamacedo86 @StephenCoady you wouldn't need to worry about this at all if you just use `model.findOrCreate` because it would be a single line of code.

Reply:
Hmm, it's kind of two unrelated things: 1) updating the resolver and 2) possibly creating a subscription. But if we can reduce it to one line using `findOrCreate` then it should be fine to keep it in `upsertResolver`.